### PR TITLE
i18n(ko-KR): add `caisy.mdx` & `sanity.mdx` & `sitecore.mdx`.

### PR DESCRIPTION
**Pull request by yeonjulee1005**
#### Description (required)

- add `caisy.mdx`
- add`sanity.mdx`
- add`sitecore.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by VoxelMC — APPROVED** [review]
Wow! Thank you for updating these pages!
`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

**Review by dreyfus92 — APPROVED** [review]
Thank you for handling this @yeonjulee1005 and thank you for the review @VoxelMC , LGTM ✅

File: src/content/docs/ko/guides/cms/sitecore.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  ```shell
+  npx @astro-sitecore-jss/create-astro-sitecore-jss@latest
+  ```
+3. 터미널의 지침에 따라 프로젝트를 생성하세요.
+
+## 커뮤니티 자료
+
+- [Astro 용 Sitecore Javascript 소프트웨어 개발 키트](https://github.com/exdst/jss-astro-public)
+- [Astro를 사용한 Sitecore 소개](https://exdst.com/posts/20231002-sitecore-astro)
+- [첫 번째 Sitecore Astro 프로젝트 시작하기](https://exdst.com/posts/20240103-first-sitecore-astro-project)
```
Comment (sarah11918) [inline src/content/docs/ko/guides/cms/sitecore.mdx:25]:
```suggestion
- [첫 번째 Sitecore Astro 프로젝트 시작하기](https://exdst.com/posts/20240103-first-sitecore-astro-project)

```

**Comment by sarah11918** [comment]
Hi @yeonjulee1005 , thank you for these translations!  💪 

In order for us to merge them, this PR needs to be set so that "maintainers can make changes". There should be a checkbox you can fill in to give us this permission. This allows us to update your branch so we can ensure there are no other conflicts when docs has been updated since you made your PR.

If you have trouble enabling this setting, maybe @jsparkdev can give you some advice, as he works on a lot of our Korean translations!  Please let us know when you have added that setting to this PR, and make sure to do so on all future PRs. Thank you!

File: src/content/docs/ko/guides/cms/caisy.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+description: 'Caisy를 CMS로 사용하여 Astro 프로젝트에 콘텐츠를 추가하세요.'
+type: cms
+service: Caisy
+i18nReady: false
+stub: true
+---
+
+[Caisy](https://caisy.io/)는 콘텐츠에 엑세스하기 위해 GraphQL API를 노출하는 헤드리스 CMS입니다.
+
+## Astro 와 함께 Caisy CMS 사용하기
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/caisy.mdx:12]:
```suggestion
## Astro와 함께 Caisy CMS 사용하기
```

File: src/content/docs/ko/guides/cms/caisy.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+const post = gqlResponse?.allBlogArticle?.edges?.[0]?.node;
+---
+<h1>{post.title}</h1>
+<RichTextRenderer node={post.text.json} />
+```
+
+## 공식 자료
+
+- [GitHub](https://github.com/caisy-io/caisy-example-astro) 또는 [StackBlitz](https://stackblitz.com/github/caisy-io/caisy-example-astro?file=src%2Fpages%2Fblog%2F%5B...slug%5D.astro) Caisy + Astro 예시를 확인해 보세요.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/caisy.mdx:59]:
```suggestion
- [GitHub](https://github.com/caisy-io/caisy-example-astro) 또는 [StackBlitz](https://stackblitz.com/github/caisy-io/caisy-example-astro?file=src%2Fpages%2Fblog%2F%5B...slug%5D.astro)에서 Caisy + Astro 예시를 확인해 보세요.
```

File: src/content/docs/ko/guides/cms/caisy.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- [초안 모드](https://caisy.io/developer/docs/external-api/localization-and-preview#preview-mode-15) 및 여러 [언어](https://caisy.io/developer/docs/external-api/localization-and-preview#localization-in-a-graphql-query-8)에서 문서를 검색하세요.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/caisy.mdx:60]:
```suggestion
- [초안 모드](https://caisy.io/developer/docs/external-api/localization-and-preview#preview-mode-15) 및 여러 [언어](https://caisy.io/developer/docs/external-api/localization-and-preview#localization-in-a-graphql-query-8)로 문서를 쿼리하세요.
```

File: src/content/docs/ko/guides/cms/caisy.mdx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- 많은 수의 문서를 조회하려면 [pagination](https://caisy.io/developer/docs/external-api/queries-pagination)를 사용하세요.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/caisy.mdx:61]:
```suggestion
- 많은 수의 문서를 조회하려면 [pagination](https://caisy.io/developer/docs/external-api/queries-pagination)을 사용하세요.
```

File: src/content/docs/ko/guides/cms/sitecore.mdx [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+title: Sitecore Experience Manager & Astro
+description: 'Sitecore를 CMS로 사용하여 프로젝트에 콘텐츠를 추가하세요.'
+type: cms
+stub: true
+service: Sitecore XM
+i18nReady: true
+---
+
+[Sitecore 경험 관리자 (XM)](https://www.sitecore.com/products/experience-manager)는 ASP.NET을 기반으로 구축된 엔터프라이즈급 콘텐츠 관리 시스템입니다.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/sitecore.mdx:10]:
```suggestion
[Sitecore Experience Manager (XM)](https://www.sitecore.com/products/experience-manager)는 ASP.NET을 기반으로 구축된 엔터프라이즈급 콘텐츠 관리 시스템입니다.
```

서비스 명칭은 번역할 필요가 없을 것 같습니다 :)

File: src/content/docs/ko/guides/cms/sitecore.mdx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+## 시작하기
+
+1. [Sitecore 헤드리스 웹사이트 만들기](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html)는 Sitecore의 공식문서를 따르세요.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/sitecore.mdx:14]:
```suggestion
1. Sitcore의 공식 문서에 따라 [Sitecore Headless 웹사이트를 만드세요](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html).
```

File: src/content/docs/ko/guides/cms/sitecore.mdx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+2. 터미널에서 다음의 프로젝트 초기 명령어를 실행하세요.
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/sitecore.mdx:15]:
```suggestion
2. 터미널에서 다음 프로젝트 초기 명령어를 실행하세요. 
```

File: src/content/docs/ko/guides/cms/sitecore.mdx [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+1. [Sitecore 헤드리스 웹사이트 만들기](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html)는 Sitecore의 공식문서를 따르세요.
+2. 터미널에서 다음의 프로젝트 초기 명령어를 실행하세요. 
+  ```shell
+  npx @astro-sitecore-jss/create-astro-sitecore-jss@latest
+  ```
+3. 터미널의 지침에 따라 프로젝트를 생성하세요.
+
+## 커뮤니티 자료
+
+- [Astro 용 Sitecore Javascript 소프트웨어 개발 키트](https://github.com/exdst/jss-astro-public)
```
Comment (jsparkdev) [inline src/content/docs/ko/guides/cms/sitecore.mdx:23]:
```suggestion
- Github의 [Astro용 Sitecore JavaScript 소프트웨어 개발 키트](https://github.com/exdst/jss-astro-public)
```

**Review by jsparkdev — COMMENTED** [review]
수고하셨습니다!

**Comment by yeonjulee1005** [comment]
@sarah11918 I updated Settings as you mention! :)

**Review by jsparkdev — APPROVED** [review]
수고하셨습니다.

LGTM

**Review by dreyfus92 — COMMENTED** [review]
Thanks everybody 😁

**Pull request merged**